### PR TITLE
Add .woisping command

### PR DESCRIPTION
**Pull request by diewellenlaenge**
With `.woisping` every member of that role can use this command to ping all members of the `Woisgang` role.
Can only be used every now and then (see config, default every 2 hours).
Mods can override the last ping time. Therefore it's advisable that mods always use `.woisping` instead of directly using `@Woisgang` to set the last ping time accordingly to avoid spamming by users.

Don't forget to update the `config.json`.

@tobi6112

### CODE REVIEW

File: src/commands/woisping.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/**
+ * Allows usage of @Woisgang mention for people having that role assigned
+ *
+ * @param {import("discord.js").Client} client
+ * @param {import("discord.js").Message} message
+ * @param {Array} args
+ * @param {Function} callback
+ * @returns {Promise<Function>} callback
+ */
+exports.run = async(client, message, args, callback) => {
```
Comment (NullDev) [inline src/commands/woisping.js:28]: why `async`? We're not `await`-ing anything in this block

**Comment by tobi6112** [comment]
Honor

File: src/commands/woisping.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+// ================================= //
+// = Copyright (c) diewellenlaenge = //
+// ================================= //
+
+/**
+ * @typedef {import("discord.js").TextChannel} TC
```
Comment (NullDev) [inline src/commands/woisping.js:8]: This `typedef` isn't used anywhere in this file

**Review by NullDev — APPROVED** [review]

File: src/commands/woisping.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        log.warn(`User "${message.author.tag}" (${message.author}) tried command "${config.bot_settings.prefix.command_prefix}woisping" and was denied`);
+
+        return callback(
+            `Tut mir leid, ${message.author}. Du hast nicht genügend Rechte um dieses Command zu verwenden =(`
+        );
+    }
+
+    let woisgangRole = message.guild.roles.cache.get(config.ids.woisgang_role_id);
+
+    if (woisgangRole === undefined) {
```
Comment (tobi6112) [inline src/commands/woisping.js:39]: Why not `if(!woisgangRole)`?

**Pull request merged**